packages/SwingSet/test/upgrade/vat-ulrik-1.js

```diff
@@ -167,7 +168,78 @@ export const buildRootObject = (_vatPowers, vatParameters, baggage) => {
     returnEternalPromise: () => p2,
 
     makeLostKind: () => {
-      makeKindHandle('unhandled', []);
+      makeKindHandle('unhandled');
+    },
+
+    makeMultiKind: mode => {
+      const mkh = makeKindHandle('multi');
+      baggage.init('mkh', mkh);
+      switch (mode) {
+        case 's2mFacetiousnessMismatch': {
+          // upgrade should fail
+          defineDurableKind(mkh, () => ({}), {
+            fooMethod: () => 1,
```

Comment:
nit: since we aren't looking at method names at all, this test could be a lot shorter to read if every facet had the same method, and they were all defined ahead of time in a single `const facet = { method: () => 1 };`. Also, `const emptyInit = () => ({});`. Then e.g. `facetCountMismatch` would squish into:
```js
  defineDurableKindMulti(mkh, emptyInit, {
    foo: facet,
    bar: facet,
  });
```

also nit: can we use `handle` for the local variable name instead of `mkh`?